lib/reporting/reports/customers/addresses.rb

```diff
@@ -31,9 +32,13 @@ def columns
             shipping_method: proc { |orders| orders.first.shipping_method&.name },
             total_orders: proc { |orders| orders.count },
             total_incl_tax: proc { |orders| orders.sum(&:total) },
+            last_completed_order_date: proc { |orders|
+                                         orders.max_by(&:completed_at)&.completed_at&.to_date
```

Comment:
Maybe you should move this to it's on method, it should help with the Cyclimatic Complexity

Reply:
Yes! 8fea985f2d